kornia/geometry/liegroup/_utils.py

```diff
@@ -0,0 +1,3 @@
+def squared_norm(m):
+    assert m.shape[-1] == 3
+    return m.pow(2).sum(-1, True)
```

Comment:
@cjpurackal i believe you will need a batched version of this
https://github.com/kornia/kornia/blob/master/kornia/geometry/quaternion.py#L319